File: C++/Easy/Accepted/minimum-operations-to-make-the-array-increasing_5.cpp
class Solution {
public:
    int minOperations(vector<int>& nums) {
        
        int ans = 0;
        int prev = nums[0];
        for(int i = 1; i < nums.size(); ++i){
            if(nums[i] <= prev){
                ans += prev - nums[i] + 1;
                prev++;
            }else{
                prev = nums[i];
            }
        }
        return ans;
    }
};

File: Java/Easy/Accepted/minimum-operations-to-make-the-array-increasing_5.java
class Solution {
    public int minOperations(int[] nums) {
        
        int res = 0;
        int prev = nums[0];
        for (int i = 1; i < nums.length; i++) {
            if (nums[i] <= prev) {
                res += prev + 1 - nums[i];
                prev++;
            } else {
                prev = nums[i];
            }
        }
        return res;
    }
}